src/status_im/ui/screens/status/views.cljs

```diff
@@ -33,7 +33,9 @@
   (let [width (reagent/atom nil)]
     (fn [uri show-close?]
       [react/view {:style               {:width         @width
+                                         :align-items   :center
                                          :height        image-max-dimension
+                                         :max-width     :100%
```

Comment:
could you please elaborate on how this will work?

Reply:
The modification here is to the container view which contains the image component.
problem : In case of horizontally long images the image width would be bigger than the device width and the image would overflow outside of the view.
Solution : set the container view to have a max-width of 100% which ensures that no matter what is the width of the image (which is in the child component) in question it would never overflow because of the max-width property.

Screenshot before fix :

 
![sceenshot-before-fix](https://user-images.githubusercontent.com/64726664/160056412-1b75fc6b-2410-46f5-a114-c39d6f2f65e1.jpg)



Screenshot after fix : 



![screenshot-after-fix](https://user-images.githubusercontent.com/64726664/160056441-588bf5f7-676b-4890-bb54-974f798ff26b.jpg)

Reply:
oh i see, but now the image is cut on the left?

Reply:
Its center aligned actually, Some images can be reallly long horizontally and to accommodate them there were 2 options : 
1) Either reduce the height to match the aspect ratio ( because of long length )
OR
2) Center align them with a fixed height
The second one seemed more readable to me given our constraints of device width

Reply:
Also when you tap on these images it opens them in a modal view in full